src/Cryptol/TypeCheck/PP.hs

```diff
@@ -6,7 +6,7 @@
 -- Stability   :  provisional
 -- Portability :  portable
 
-{-# LANGUAGE Safe #-}
+{-# LANGUAGE Trustworthy #-}
```

Comment:
In this, and a lot of other places, there is a bunch of noise messing around with Safe annotations and such.  Is all that really necessary?

Reply:
I was getting a bunch of warnings from HLS, so that's why I did it